x/ref/runtime/internal/flow/conn/auth.go

```diff
@@ -11,7 +11,7 @@ import (
 	"time"
 
 	"golang.org/x/crypto/nacl/box"
-	"v.io/v23"
+	v23 "v.io/v23"
```

Comment:
I'm not sure why "v23" alias is needed.

Reply:
vXYZ in package import paths specify semantic import versioning for the package, and are stripped when inferring the package name from the import path. So we need to specify the package name manually.

https://blog.golang.org/versioning-proposal